Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3-alpine
+
+MAINTAINER Jakub Dorňák <jakub.dornak@misli.cz>
```

Comment:
`MAINTAINER` is deprecated [according to the Docker docs](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). You should use `LABEL maintainer="..."` instead.